pokemongo_bot/cell_workers/sniper.py

```diff
@@ -362,8 +366,7 @@ def work(self):
 
             if targets:
                 # Order the targets (descending)
-                for attr in self.order:
-                    targets.sort(key=lambda pokemon: pokemon[attr], reverse=True)
+                targets = sorted(targets, key=itemgetter(*self.order), reverse=True)
```

Comment:
I know. Thats what I meant. Although, the final/merged list will have around 30~50 items, and if you have a max of 5 ordering times, that should be an iteration of 150~250 times, which is worthless. I'll accept it anyways.